addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1267,6 +1267,15 @@ options.registry.SnippetMove = options.Class.extend({
     },
 });
 
+/**
+ * Test option do not merge.
+ */
+options.registry.ImagepickerTest = options.Class.extend({
+    image: function (previewMode, widgetValue, params) {
+        console.log(...arguments);
+    }
+});
```

Comment:
My guess is we can remove this now :)

Reply:
Just left it in there to make it easier to make sure the data you wanted was passed to the method. Removing this now then :)